docs/user/technical-reference/03-10-architecture-updates.md

```diff
@@ -0,0 +1,20 @@
+# Serverless Architecture Updates
+
+## Changes
+
+- The internal Docker Registry is no longer part of the Serverless module. Instead, the Docker Registry is now a separate, standalone module.
+- There is no longer a build job for Functions. Instead, a base image is used, which mounts the required dependencies dynamically.
+- Libraries and dependencies are downloaded at the start of the Function's execution. This means that each replica of the Function can potentially use a different version of the dependencies.
+- Function code is now injected directly into the runtime Pod, eliminating the need for pre-built images.
+
+## Benefits
+
+- **Simplified architecture**: By separating the Docker Registry into its own module, the Serverless module is now more lightweight and easier to manage.
+- **Faster deployment**: The removal of the build job reduces the time required to deploy Functions.
+- **Dynamic dependency resolution**: Dependencies are resolved at runtime, allowing for more flexibility in managing library versions.
+- **Improved flexibility**: Injecting Function code into the runtime Pod simplifies the deployment process and reduces image management overhead.
+
```

Comment:
- eliminates the need to for serverless to acquire computational resources from worker nodes to build the image
 - improved security:  by eliminating build jobs functions can run in namespaces with more restrictive pod security levels enabled